src/utils/endretAvUtil.ts

```diff
@@ -1,14 +1,18 @@
+export const FOLKEREGISTERET = 'SKD';
+
 export function endretAvTekst(rawString: string): string {
     if (endretAvBruker(rawString) || endretIPSelv(rawString)) {
         return 'av bruker';
     } else if (endretIFagsystem(rawString) || endretIPesys(rawString) ) {
         return 'av NAV';
     } else if (rawString.match('AAA2101, SKD')) {
         return 'av Skatteetaten';
-    } else if (rawString.match('SKD')) {
-        return 'i Folkeregisteret';
     } else {
-        return rawString;
+        if (rawString.match(FOLKEREGISTERET)) {
```

Comment:
Noen spesiell grunn til at du nøsta dette inn i else? Istedetfor i else if slik som resten?

Reply:
1. Fordi jeg bruker variablen et annet sted.
2. Trekke den ut i en konstant for bedre forståelese? 🤔